Doc/library/struct.rst

```diff
@@ -370,26 +378,24 @@ the result in a named tuple::
     Student(name=b'raymond   ', serialnum=4658, school=264, gradelevel=8)
 
 The ordering of format characters may have an impact on size since the padding
-needed to satisfy alignment requirements is different::
-
-    >>> pack('ci', b'*', 0x12131415)
-    b'*\x00\x00\x00\x12\x13\x14\x15'
-    >>> pack('ic', 0x12131415, b'*')
-    b'\x12\x13\x14\x15*'
-    >>> calcsize('ci')
+needed to satisfy alignment requirements is different. In this
+example, the output was produced on a little endian machine::
+
+    >>> pack('@ci', b'*', 0x12131415)
+    b'*\x00\x00\x00\x15\x14\x13\x12'
+    >>> pack('@ic', 0x12131415, b'*')
+    b'\x15\x14\x13\x12*'
+    >>> calcsize('@ci')
     8
-    >>> calcsize('ic')
+    >>> calcsize('@ic')
     5
 
 The following format ``'llh0l'`` specifies two pad bytes at the end, assuming
 longs are aligned on 4-byte boundaries::
 
-    >>> pack('llh0l', 1, 2, 3)
+    >>> pack('@llh0l', 1, 2, 3)
     b'\x00\x00\x00\x01\x00\x00\x00\x02\x00\x03\x00\x00'
 
-This only works when native size and alignment are in effect; standard size and
```

Comment:
I'd keep this remark - it is important. Maybe "This padding behaviour is because native size and alignment are in effect" for the first part instead though?

Reply:
The old sentence was bad.  Everything the reader needs to know is said in the preamble before this example.

Reply:
I'm going to leave this as-is. It's not clear to me which remark Cameron is referring to. Unlike most GitHub diffs which provide too little context, this seems to have too much.